src/main/java/com/alex/eshop/eshop/Repository/ItemRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.alex.eshop.eshop.Repository;
+
+import com.alex.eshop.eshop.Entity.Item;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+@EnableJpaRepositories
```

Comment:
Not sure what exactly `@EnableJpaRepositories` annotation is doing, but we don't need it